bootstrap/_bootstrap_common.py

```
@@ -0,0 +1,257 @@
+from __future__ import annotations
+
+import os
+import re
+import shutil
+import socket
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List
```

Ruff (0.15.10)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)